Add issue and PR templates (#9297)
astropy/astropy: .github/ISSUE_TEMPLATE/bug_report.md

https://github.com/astropy/astropy/blob/master/CODE_OF_CONDUCT.md . -->

<!-- Please have a search on our GitHub repository to see if a similar
issue has already been posted.
If a similar issue is closed, have a quick look to see if you are satisfied
by the resolution.
If not please go ahead and open an issue! -->

<!-- Please check that the development version still produces the same bug.
You can install development version with
pip install git+https://github.com/astropy/astropy
command. -->

### Description
<!-- Provide a general description of the bug. -->

### Expected behavior
<!-- What did you expect to happen. -->

### Actual behavior
<!-- What actually happened. -->
<!-- Was the output confusing or poorly described? -->

### Steps to Reproduce
<!-- Ideally a code example could be provided so we can run it ourselves. -->
<!-- If you are pasting code, use tripe backticks (```) around
your code snippet. -->
<!-- If necessary, sanitize your screen output to be pasted so you do not
reveal secrets like tokens and passwords. -->

1. [First Step]
2. [Second Step]
3. [and so on...]

```python
# Put your Python code snippet here.
```

### System Details

> <!-- Even if you do not think this is necessary, it is still nice to have.

Reviewer: ```suggestion
<!-- Even if you do not think this is necessary, it is useful information for the maintainers.
```

<!-- Please have a search on our GitHub repository to see if a similar
issue has already been posted.
If a similar issue is closed, have a quick look to see if you are satisfied
by the resolution.
If not please go ahead and open an issue! -->

<!-- Please check that the development version still produces the same bug.
You can install development version with
pip install git+https://github.com/astropy/astropy
command. -->

### Description
<!-- Provide a general description of the bug. -->

### Expected behavior
<!-- What did you expect to happen. -->

### Actual behavior
<!-- What actually happened. -->
<!-- Was the output confusing or poorly described? -->

### Steps to Reproduce
<!-- Ideally a code example could be provided so we can run it ourselves. -->
<!-- If you are pasting code, use tripe backticks (```) around
your code snippet. -->
<!-- If necessary, sanitize your screen output to be pasted so you do not
reveal secrets like tokens and passwords. -->

1. [First Step]
2. [Second Step]
3. [and so on...]

```python
# Put your Python code snippet here.
```

### System Details
<!-- Even if you do not think this is necessary, it is still nice to have. -->

> <!-- import astropy; print(astropy.__version__) -->

Reviewer: I like at sklearn land that they have a copy paste-able block for this:
```
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import astropy; print("Astropy", astropy.__version__)
```

Author: Done.
